src/ports/catalog/queries.ts

```diff
@@ -589,7 +604,7 @@ const getTradesJoin = () => {
           FROM unified_trades
             WHERE status = 'open'
             GROUP BY contract_address_sent, assets -> 'sent' ->> 'item_id'
-          ) AS offchain_orders ON offchain_orders.contract_address_sent = items.collection_id AND offchain_orders.item_id = items.blockchain_id::text AND items.available > 0
+          ) AS offchain_orders ON offchain_orders.contract_address_sent = items.collection_id AND offchain_orders.item_id = items.blockchain_id::text -- AND items.available > 0
```

Comment:
is the last part of the query still required? `AND items.available > 0`

Reply:
not needed anymore! thanks!